Libraries/LibWeb/WebGL/WebGL2RenderingContextBase.idl

```diff
@@ -426,8 +426,8 @@ interface mixin WebGL2RenderingContextBase {
     [FIXME] undefined uniformBlockBinding(WebGLProgram program, GLuint uniformBlockIndex, GLuint uniformBlockBinding);
 
     // Vertex Array Objects
-    [FIXME] WebGLVertexArrayObject createVertexArray();
-    [FIXME] undefined deleteVertexArray(WebGLVertexArrayObject? vertexArray);
-    [FIXME] GLboolean isVertexArray(WebGLVertexArrayObject? vertexArray); // [WebGLHandlesContextLoss]
-    [FIXME] undefined bindVertexArray(WebGLVertexArrayObject? array);
+    WebGLVertexArrayObject createVertexArray();
+    undefined deleteVertexArray(WebGLVertexArrayObject? vertexArray);
+    GLboolean isVertexArray(WebGLVertexArrayObject? vertexArray); // [WebGLHandlesContextLoss]
```

Comment:
the comment on WebGL 1 context base said that we handle context loss in the C++ platform object. is that not re-implemented since the new angle impl?

Reply:
yeah, there is no context loss handling at all yet.